<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alejandro Dini</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="columnaIzquierda">
      <item>
       <widget class="QLabel" name="labelTabla">
        <property name="text">
         <string>PÃ³lizas de Auto Vigentes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tablaPolizas"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="columnaDerecha">
      <item>
       <widget class="QPushButton" name="nuevaButton">
        <property name="text">
         <string>Nueva</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editarButton">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cambiarEstadoButton">
        <property name="text">
         <string>Cambiar Estado</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Abandonar el programa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
   <property name="toolTip">
    <string>Acerca de este TP</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
